src/Game/Game.js

```diff
@@ -1,28 +1,160 @@
-import {useState} from "react";
```

Comment:
If I were to rewrite this file, I would have a top level function called turn or something that went through each step and called a function so you could see all of the functions involved in a "turn" in their proper sequence